autometa/common/kmers.py

```diff
@@ -369,9 +365,10 @@ def embed(kmers=None, embedded=None, n_components=2, do_pca=True, pca_dimensions
         Reduce k-mer frequencies dimensions to `pca_dimensions` (the default is 50).
         If None, will estimate based on
     method : str, optional
-        embedding method to use (the default is 'UMAP').
+        embedding method to use (the default is 'umap').
```

Comment:
Wasn't the default method supposed to be 'TSNE' so that the refactored code behaves like Autometa 1.0?